plugins/wu_palmer.py

```diff
@@ -414,10 +420,8 @@ def add_similarities_for_entities(
                 sim = wu_palmer_cache.calculate_similarity(tax_name, val1, val2)
 
             similarities[(val1, val2)] = {
-                "ID": str(val1) + "_" + str(val2),
-                "href": "",
-                "value_1": val1,
-                "value_2": val2,
+                "source": val1,
```

Comment:
these values were sorted. This is ok for calculating the similarity but we may want to preserve the order to correctly determine which value was from which entity later.